devicetypes/widomsrl/widom-smart-dry-contact.src/widom-smart-dry-contact.groovy

```diff
@@ -0,0 +1,340 @@
+/**
+ *  Widom Smart DRY contact
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "WiDom Smart Dry Contact", namespace: "WiDomsrl", author: "WiDom srl", ocfDeviceType: "oic.d.switch", mnmn: "SmartThings", vid: "generic-switch") {
+		capability "Actuator"
+		capability "Switch"
+		capability "Configuration"
+		capability "Health Check"
+
+		fingerprint mfr: "0149", prod: "1214", model: "0900", deviceJoinName: "WiDom Dry Contact" // WiDom Dry Contact allow to turn On/Off a load with indipendent power supply.
```

Comment:
(this remark was added previously, but I think You missed it)
Please add deviceJoinName "WiDom Switch" in this fingerprint and add Raw Description in a comment after the fingerprint so it would look like:
fingerprint mfr: "0149", prod: "1214", model: "0900", deviceJoinName "WiDom Switch" // Raw Description (paste the raw description here)

(Raw description can be found at the Smartthings graphapi in device details)

Reply:
Raw Description fixed on 1st April 2021